frontend/packages/pipelines-plugin/src/topology/pipelinesTopologyPlugin.ts

```diff
@@ -31,16 +29,4 @@ export const pipelinesTopologyPlugin: Plugin<PipelineTopologyConsumedExtensions>
       required: [FLAG_OPENSHIFT_PIPELINE],
```

Comment:
I think we should move this `type: 'Topology/DataModelFactory'` to dynamic plugin as well and remove this `PipelineTopologyPlugin` from plugin.tsx .